index.html

```diff
@@ -0,0 +1,250 @@
+            </ul>
+        </div>
+        <!--*******Desktop Navigation Menu******-->
+        <nav class="nav-menu-desktop">
+         <ul class="list">
+                <li><a class="header-icon" href="#"><i class="fa-brands fa-facebook-f"></i></i></a></li>
+                <li><a class="header-icon" href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li>English</li>
+            <li>My page</li>
+            <li>Log out</li>
+            </ul>
+        </nav>
+        <!--*******Desktop Navigation Menu******-->
+            <nav class="nav-menu-dekhtop2">
+                   <ul class="list2">
+                    <li class="summit-logo"><a href="#"><img class="logolink" src="./images/websummitlogo.png" alt="WebLogo"></a></li>
+                        <li class="nav-links"><a href="about.html">About</a></li>
+                        <li class="nav-links"><a href="#">Exhibition</a></li>
+                        <li class="nav-links"><a href="#">Join</a></li>
+                        <li class="nav-links"><a href="#">Sponsor</a></li>
+                        <li  class="nav-links"><a href="#">Schedules</a></li>
+                        <li class="nav-links"><a class="border-link" href="#program">PROGRAMS</a></li>
+                    
+                    </ul>
+            </nav>
+       </header> 
+<article>
+    <!--*******Mobile Headline Section******-->
+<div class="headline">
+    <h3 class="topheadingh3">Hello Tech World!</h3>
+    <div>
+        <h1 class="topheadingh1">Web Global Annual Summit 2023</h1>
+    </div>
+    <div class="paradiv"><p class="headlinepara">“Where the future goes to be born” <br>
+        - The Atlantic <br><br>
+        Web Summit will bring together 70,000+ people, and the companies redefining the tech industry. <br><br>
+
+        See you in Lisbon this November.
+    </p></div>
+    <div class="datediv">
+        <h2 class="dateheading">13-11-2023(Mon) ~ (Thu)</h2>
+        <p class="locationpara">
+            @ALTICE ARENA & FIL, LISBON
+        </p>
+      </div>
+</div>
+<!--*******Desktop Headline Section******-->
+<div class="headlinedesktop">
+    <div class="topdesktop">
+        <h3 class="topheadingdesktop">"Hello Tech World!"</h3>
+    </div>
+    <div class="topheadingdesktopdiv">
+        <h1 class="topheadingh1desktop">Web Global Annual <br> Summit  2023</h1>
+        <div class="paradivdesktop"><p class="headlineparadesktop">“Where the future goes to be born” <br>
+            - The Atlantic <br><br>
+            Web Summit will bring together 70,000+ people, and the companies redefining the tech industry. <br><br>
+    
+            See you in Lisbon this November.
+        </p></div>
+        <div class="datedivdesktop">
+            <h2 class="dateheadingdesktop">13-11-2023(Mon) ~ (Thu)</h2>
+            <p class="locationparadesktop">
+                @ALTICE ARENA & FIL, LISBON
+            </p>
+          </div>
+    </div>
+    </div>
+</article>
```

Comment:
- [ ]  As previous reviewer suggested, It seemed that you've used Article tag in place of section tag. maybe to provide with enough details, check out these codes, they will help you to understand how you were supposed to structure your codes properly.  
consider these changes in both `index.html` and `about.html` files
[semantic sample codes
](https://gist.github.com/ishimwezachee/27635e614feba10b49df5b172915e8aa)

Reply:
I have replaced the article with Section tag please refer to the screen shot below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/219655564-e968a142-b528-4e27-8231-4bd0ced18695.png)